clientv3/maintenance.go

```diff
@@ -50,6 +51,11 @@ type Maintenance interface {
 	// Status gets the status of the endpoint.
 	Status(ctx context.Context, endpoint string) (*StatusResponse, error)
 
+	// HashKV returns the state of backend database at the time of the RPC.
+	// If revision is non-zero, hash is computed only in key bucket and keys at or below
```

Comment:
clientv3 Compact and the rpc.proto Compact have different comments too; the difference here seems to be similar. Although to be closer to clientv3 Compact it should talk about KV history instead of a "backend database".

Duplicated comments (/anything) can easily get out of sync and I think there are too many exceptions to using the RPC comment directly to justify tooling to keep it synced up.